src/_P101_WakeOnLan.ino

```diff
@@ -95,21 +98,17 @@ WakeOnLan WOL(udp);
 
 // ************************************************************************************************
 // Local Functions
-bool validateIp(String ipStr);
-bool validateMac(String macStr);
-bool validatePort(String portStr);
+uint8_t safeName(taskIndex_t index);
+bool    validateIp(String ipStr);
+bool    validateMac(String macStr);
+bool    validatePort(String portStr);
```

Comment:
Please use `const String&` as function argument.
By giving the argument as a const reference, you prevent the content to be copied.
As it is now, you copy the content into a new object.

Reply:
Nice! New file uploaded.
- Thomas